test/optimizer/applyImportAndExportNameChanges2-output.js

```diff
@@ -275,4 +275,4 @@ WebAssembly.instantiate(Module["wasm"], imports).then(output => {
  dynCall_iiii = asm["p"];
  initRuntime(asm);
  ready();
-});
+}));
```

Comment:
Why are these needed?

Reply:
They are not needed, but its seems like acorn is producing them now.. I guess its just a slight behaviour change.

As you can see, no actual code size tests regressed so I guess closure is able just erase these.

If we can we would need to investigate why acorn started doing this.. and see if we can disable it.

Reply:
Do you think we care about these braces?

Reply:
Yeah, I guess closure fixes this up. Odd terser slightly regressed here but probably this is related to some bugfix for them...